mllib/src/test/scala/org/apache/spark/ml/feature/VectorIndexerSuite.scala

```diff
@@ -283,7 +283,9 @@ class VectorIndexerSuite extends MLTest with DefaultReadWriteTest with Logging {
         points.zip(rows.map(_(0))).foreach {
           case (orig: SparseVector, indexed: SparseVector) =>
             assert(orig.indices.length == indexed.indices.length)
-          case _ => throw new UnknownError("Unit test has a bug in it.") // should never happen
+          case _ =>
+            // should never happen
+            throw new IllegalAccessException("Unit test has a bug in it.")
```

Comment:
Just `fail()` here? or at least not `IllegalAccessException`

Reply:
Thanks, done in a4f49ce by just `fail` here.